app/assets/stylesheets/pin-index.css.scss

```diff
@@ -0,0 +1,130 @@
+$primary-background-color: rgba(237, 242, 242, 0.8);
+
+
+#container{
+  background-color: $primary-background-color;
+}
+
+hr{
```

Comment:
DOnt think we will really have an hr, this is just to divide the partial, there's header before the hr. that's all